### DISCUSSION TITLE
zh-cn/posts/my-new-blog/

### DISCUSSION THREAD

**Discussion by giscus** [Clouder0/ClouderBlogComment#29] [Announcements]
# zh-cn/posts/my-new-blog/

最近投入了大量摸鱼时间重构博客。现在大概告一段落了，向大家介绍一下整体的技术选型和具体实现的简要思路。

https://clouder0.com/zh-cn/posts/my-new-blog/

**Comment by Clouder0** [comment]
不过现在评论区的一个问题是按 slug 分评论，那么中英文实际上是分开的...不过似乎可以当成 Feature.

**Comment by longlin10086** [comment]
呜呜看不懂awa🥹
